main.py

```diff
@@ -1,10 +1,52 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        data = json.load(file)
+
+    for player_name, player_info in data.items():
+        race = player_info.get("race")
+        skills = player_info.get("race")["skills"]
+        guild = player_info.get("guild")
+
+        if Race.objects.filter(name=race["name"]).exists():
+            race = Race.objects.get(name=race["name"])
+        else:
+            race = Race.objects.create(name=race["name"],
+                                       description=race["description"])
+
+        for skill in skills:
+            if not Skill.objects.filter(name=skill["name"]).exists():
+                skill = Skill.objects.create(
+                    name=skill.get("name"),
+                    bonus=skill.get("bonus"),
+                    race=race)
+            else:
+                Skill.objects.get(name=skill["name"])
```

Comment:
Bonus for these fields, you can simply use get_or_create instead whole check, but it's just a suggestion:
[https://www.queworx.com/django/django-get_or_create/](url)